pkgs/by-name/ls/lsp-ai/package.nix

```diff
@@ -0,0 +1,80 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  cmake,
+  openssl,
+  zlib,
+  perl,
+  gitUpdater,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "lsp-ai";
+  version = "0.7.0";
+  src = fetchFromGitHub {
+    owner = "SilasMarvin";
+    repo = "lsp-ai";
+    rev = "release/v${version}";
+    hash = "sha256-5chIz9uIKmwnZG1HjEjqZqoWi3aTmh51r5YJD21u2OE=";
+  };
+
+  passthru.updateScript = gitUpdater {
+    rev-prefix = "v";
+  };
+
+  checkFlags = [
+    # These integ tests require an account and network usage to work
+    "--skip=transformer_backends::open_ai::test::open_ai_completion_do_generate"
+    "--skip=transformer_backends::mistral_fim::test::mistral_fim_do_generate"
+    "--skip=transformer_backends::anthropic::test::anthropic_chat_do_generate"
+    "--skip=transformer_backends::open_ai::test::open_ai_chat_do_generate"
+    "--skip=transformer_backends::gemini::test::gemini_chat_do_generate"
+    "--skip=transformer_backends::ollama::test::ollama_chat_do_generate"
+    "--skip=transformer_backends::ollama::test::ollama_completion_do_generate"
+    "--skip=embedding_models::ollama::test::ollama_embeding"
+    "--skip=transformer_worker::tests::test_do_completion"
+    "--skip=transformer_worker::tests::test_do_generate"
+    "--skip=memory_backends::vector_store::tests::can_open_document"
+    "--skip=memory_backends::vector_store::tests::can_rename_document"
+    "--skip=memory_backends::vector_store::tests::can_build_prompt"
+    "--skip=memory_backends::vector_store::tests::can_change_document"
+    # Additional skip flags
```

Comment:
This comment doesn't really explain why these tests are actually disabled as far as I can tell, try adding a more specific reason like you did with "requires account + network usage" above.

Reply:
Added more descriptive comment.